RealmSwift/Tests/ObjectSchemaInitializationTests.swift

```diff
@@ -182,6 +182,17 @@ class ObjectSchemaInitializationTests: TestCase {
         let types = Set(schema.properties.map { $0.type })
         XCTAssertEqual(types, Set([.string, .string, .data, .date, .object, .int, .float, .double, .bool]))
     }
+    
+    func testImplicitlyUnwrappedOptionalProperties() {
```

Comment:
Ok! Not at all, just falsely believed style in `testOptionalProperties` had been introduced after this PR when I rebased.